src/main/webapp/app/shared/teams-selection/teams-selection.service.ts

```diff
@@ -23,10 +23,13 @@ export class TeamsSelectionService {
                 .do(result => {
                     this._selectedTeam = result.body || null;
                 })
-                .flatMap(result => {
+                .catch(err => {
+                    return Observable.empty();
```

Comment:
```suggestion
                    this.selectedTeam = null;
                    return Observable.empty();
```
The team should be deselected and removed from localstorage if it's not found.